strawberryfields/ops.py

```diff
@@ -601,7 +574,7 @@ def merge(self, other):
             # decompositions, which cannot be merged.
             U1 = self.p[0]
             U2 = other.p[0]
-            U = matmul(U2, U1).x
+            U = np.matmul(U2, U1)
```

Comment:
We could even go back to 
```suggestion
            U = U2 @ U1
```
now that we no longer dynamically replace the matmul function, right?

Reply:
I think so, yes.